### ISSUE TITLE
Download - Please Clarify how to add `download` link

### ISSUE THREAD

**Issue by rturk** [rohit-gohri/redocusaurus#2]
Hi this looks amazing, thanks for maiking it available.

I'm strugling to add the `dowload` option this related with this plugin?

![petstore](https://user-images.githubusercontent.com/2236950/110170008-4c36cd80-7dd8-11eb-9e14-b9776f5cf043.png)

**Comment by rohit-gohri** [comment]
I have it disabled by default (https://github.com/rohit-gohri/redocusaurus/blob/da046b5fdb66d292f3423020cf083073bf47b37f/packages/docusaurus-theme-redoc/src/theme/Redoc/Redoc.tsx#L101), will add an option to override it.

**Comment by rohit-gohri** [comment]
Released v0.3.0. Added [`redocOptions` in theme options](https://github.com/rohit-gohri/redocusaurus/tree/main/packages/docusaurus-theme-redoc#redocoptions).

You can pass it `{ hideDownloadButton: false }` and download button should be visible.

**Comment by rohit-gohri** [comment]
Full config example:
```
[
      'redocusaurus',
      {
        specs: [
          {
            specUrl: 'https://redocly.github.io/redoc/openapi.yaml',
          }
        ],
        theme: {
          primaryColor: '#1890ff',
          redocOptions: { hideDownloadButton: false },
        },
      }
    ]
```

Let me know if you face any problems.
